internal/operator/subscription.go

```diff
@@ -97,13 +98,18 @@ func (c operatorClient) ListPackageManifests(ctx context.Context, list *pkgserve
 	}
 
 	if len(filter) > 0 {
+		var notFound []string
 		for _, f := range filter {
 			for _, pkgm := range tmppkgmlist.Items {
 				if pkgm.Name == f {
 					list.Items = append(list.Items, pkgm)
+					notFound = append(notFound, f)
```

Comment:
@trgeiger This is adding all the found packages to the notFound List.

Reply:
it should be outside the if loop.

Reply:
thanks for catching that, i had that somewhere else and hastily moved it, will fix

Reply:
or rather, i had it as a boolean and changed it to a list, but having the list where it is defeats the purpose like you said